### PR TITLE
LEX-1541 Remove the obsolete 'thumbnail URL' feature.

### PR DESCRIPTION
**Pull request by ynx**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ install:
 
   # Install drupal default profile and enable required modules.
   - /usr/bin/env PHP_OPTIONS="-d sendmail_path=`which true`" drush --verbose site-install --db-url=mysql://root:@127.0.0.1/drupal --yes
+  - composer install --dev
```
Comment (nickveenhof) [inline .travis.yml:42]: Unrelated change?

File: config/schema/acquia_lift.settings.yml [hunk 2]
```diff
@@ -98,19 +98,6 @@ acquia_lift.settings:
               type:
                 type: string
                 label: 'Item Type this is mapped to.'
-    thumbnail:
```
Comment (nickveenhof) [inline config/schema/acquia_lift.settings.yml:101]: I think there needs to be some upgrade function that removes this config no? What if it is populated? Will this work?

File: .travis.yml (hunk 1 above)
Comment (ynx) [reply .travis.yml:42]: The branch [failed on travis out of blue](https://travis-ci.org/acquia/acquia_lift/builds/176609525). Adding this line resolved the issue. I think we should keep it.

File: config/schema/acquia_lift.settings.yml (hunk 2 above)
Comment (ynx) [reply config/schema/acquia_lift.settings.yml:101]: D8 continues to work with obsolete config variables, be it a full obsolete variable or a partial obsolete within a variable. It has happened a few times so I know this for sure now.

I thought about writing an update hook, but then, I thought why pest our customers for no benefit? So I didn't. =D

Comment (nickveenhof) [reply config/schema/acquia_lift.settings.yml:101]: Hmm, I don't fully agree here. We should not leave lingering config out there.
